data_model/src/lib.rs

```diff
@@ -495,11 +492,18 @@ model! {
         Role(Box<role::Role>),
         /// [`PermissionTokenDefinition`](`permission::token::Definition`) variant.
         PermissionTokenDefinition(Box<permission::token::Definition>),
-        /// [`Validator`](`permission::Validator`) variant.
-        Validator(Box<permission::Validator>),
         /// [`Parameter`](`parameter::Parameter`) variant.
         Parameter(Box<parameter::Parameter>),
     }
+
+    /// Sized container for all possible upgradable entities.
+    #[derive(Debug, Display, Clone, PartialEq, Eq, Hash, FromVariant, Decode, Encode, Deserialize, Serialize, IntoSchema)]
+    #[ffi_type]
+    pub enum UpgradableBox {
+        /// [`Validator`](`permission::Validator`) variant.
+        #[display(fmt = "Validator")]
+        Validator(Box<permission::Validator>),
```

Comment:
```suggestion
        Validator(permission::Validator),
```

Why would you box it?